tests/integration/linux_bridge_test.py

```diff
@@ -33,6 +33,7 @@
 from .testlib.statelib import show_only
 from .testlib.statelib import INTERFACES
 from .testlib.assertlib import assert_mac_address
+from .vlan_test import vlan_interface
```

Comment:
Please move the function to `testlib/vlan.py` so that both `linux_bridge_test.py` and `vlan_test.py` could use it.